terraform/sponsored_embargo_bucket.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_s3_bucket" "sponsored_embargo_bucket" {
+  provider = aws.sponsored
+
+  bucket = "dandiarchive-embargo"
+  # acl    = "private"
+
+  cors_rule {
```

Comment:
I think the CORS rules were originally put in place for the dandi-hub to access S3 directly. I don't think we need them for the embargo bucket, but I also don't think it hurts to have the same configuration since the access policies will restrict access anyway.